src/logger.ts

```diff
@@ -7,12 +7,19 @@ export function setVerbose(value: boolean)
     isVerbose = value;
 }
 
-export function warn(msg: string, data?: any)
+export function warn(msg: string, data?: any|TAnyDoclet)
```

Comment:
```suggestion
export function warn(msg: string, data?: any)
```

No need for the extra type there.